### PR TITLE
Add Option values to Explorer histogram data

### LINKED ISSUE #703: Allow for nullability in retrieved histogram data
There are points when loading the Block Explorer's Explorer Page where the endpoint results in an error.  This error is caused by the information in a block's header being stored before the block's payload.  A query for this information can occur between these two writes, resulting in the inability to find a payload for a given block.  When this occurs, the ExplorerHistogram population process via SQL fails as we aren't expecting a `null` value when retrieving this data.

The proposed solution is to allow for some `null` values to present in the `ExplorerHistograms` members:

https://github.com/EspressoSystems/hotshot-query-service/blob/426217cb8d0efb28b914f563dfd0e5f7df0812a6/src/data_source/storage/sql/query/explorer.rs#L407
https://github.com/EspressoSystems/hotshot-query-service/blob/426217cb8d0efb28b914f563dfd0e5f7df0812a6/src/explorer/query_data.rs#L472

### PR DESCRIPTION
**Pull request by Ayiga**
Closes #703

### This PR:

Updates the `ExplorerHistogram` fields for both `time` and `size` to have `Option` values, which can result in a `null` value coming down to the client.

### CODE REVIEW

**Review by alysiahuggins — APPROVED** [review]

File: src/data_source/storage/sql/query/explorer.rs [hunk 1]
```diff
@@ -404,32 +404,14 @@ where
                         let time: Option<i64> = row.try_get("time").map_err(|e| QueryError::Error {
                             message: format!("failed to get column time {e}"),
                         })?;
-                        let size: i32 = row.try_get("size").map_err(|e| QueryError::Error {
+                        let size: Option<i32> = row.try_get("size").map_err(|e| QueryError::Error {
```
Comment (imabdulbasit) [inline src/data_source/storage/sql/query/explorer.rs:407]: payload.size is a required field in schema so is it ever going to return None? I think we have to do left join above?

Comment (imabdulbasit) [reply src/data_source/storage/sql/query/explorer.rs:407]: nvm its nullable

**Review by imabdulbasit — APPROVED** [review]

File: src/data_source/storage/sql/query/explorer.rs (hunk 1 above)
Comment (Ayiga) [reply src/data_source/storage/sql/query/explorer.rs:407]: I actually this is this probably still a valid concern.

Comment (imabdulbasit) [reply src/data_source/storage/sql/query/explorer.rs:407]: we insert header and null payload in insert_leaf() function so no need for left join here

**Pull request merged**